app/src/main/java/com/orange/ouds/app/ui/components/checkbox/CheckboxDemoScreen.kt

```diff
@@ -84,26 +99,36 @@ fun CheckboxDemoScreen(indeterminate: Boolean = false) = DemoScreen(rememberChec
 }
 
 @Composable
-private fun CheckboxDemo(state: CheckboxDemoState, onCheckedChange: (Boolean) -> Unit) {
+private fun CheckboxDemo(state: CheckboxDemoState, onCheckedChange: (CheckboxIdentifier, Boolean) -> Unit) {
```

Comment:
As discussed together after a comment in the radio button item PR, `onCheckedChange` and `onClick` in `IndeterminateCheckboxDemo` can be removed.